Microsoft.DotNet.Interactive.Jupyter/CompleteRequestHandler.cs

```diff
@@ -45,10 +45,21 @@ private static void OnCompletionRequestCompleted(CompletionRequestCompleted comp
         {
             var command = completionRequestCompleted.Command as RequestCompletion;
 
-            var pos = SourceUtilities.ComputeReplacementStartPosition(command.Code, command.CursorPosition);
-            var reply = new CompleteReply(pos, command.CursorPosition, matches: completionRequestCompleted.CompletionList.Select(e => e.InsertText ?? e.DisplayText).ToList());
+            int startPosition, endPosition;
+            if (completionRequestCompleted.ReplacementStartIndex != null)
```

Comment:
I thought about putting this logic into `Microsoft.DotNet.Interactive/Events/CompletionRequestCompleted.cs`

but since that lives in a different project and `CompletionRequestCompleted` is more of a "data type" than something that has logic... I left the logic in this file.